### ISSUE TITLE
Website / Landing page

### ISSUE THREAD

**Issue by aha999** [Heroic-Games-Launcher/HeroicGamesLauncher#549]
It would be nice to have a basic Heroic landing page  so any articles in the future can link to it and the website would make a better visual representation of the project that could attract developers/contributors better.

A completely free and simple option is via Github Pages, that a lot of github open source projects are using, and since you're using github, it's free to host.

> Hosted directly from your GitHub repository. Just edit, push, and your changes are live.
> No databases are needed to be set up and no servers to configure

Examples of repos that use github pages: https://github.com/collections/github-pages-examples

Interesting article: https://www.toptal.com/github/unlimited-scale-web-hosting-github-pages-cloudflare

> Consider [Jekyll](https://jekyllrb.com/). You give it [Liquid ](https://jekyllrb.com/docs/liquid/)templates and Markdown content
Free Jekyll templates: https://jekyllthemes.io/free

**Comment by flavioislima** [comment]
Yes we need one but I didn't have time to start working on it yet. Maybe after 1.9 I'll find some time.

**Comment by ArturWroblewski** [comment]
Can't we go the easy way? Google domains + web page on google drive connected to this domain?

Any domain name suggestion? I can try to put one on and we will check if it works. Editing rights can be added as in any google document. What do you think?

If there is approval, we will establish a list of people who can edit and it will be quick and easy.

Only a visual suggestion - light and dark side similar to steam? what was she supposed to be next to?

Below is a link to an example page made quickly to illustrate what I'm talking about:
https://sites.google.com/view/heroicgameslauncher/epic-store

**Comment by aha999** [comment]
Isn't this the easy way ? and it's directly connected to the source, so it's easier to link things. no website on the 3rd party cloud.

**Comment by ArturWroblewski** [comment]
But running a page in github repositories is tricky. Requires code writing. 

Easier and faster to use the WYSWING graphic editor (What You See Is What You Get). This speeds up the creation of regular pages that do not need data aggregation, etc. Information pages that such a program exists and has a new version.

I admit that it is not in line with the opensource ideology.
I might be wrong. Do you know WYSWING editor that works with github?

**Comment by flavioislima** [comment]
Thanks for checking this stuff, guys. I'm focused on testing and releasing the next Heroic version and after that, we can start working on a landing page for it.
But as @ArturWroblewski said, doesn't need to be something complex. The page might need only some screenshots, descriptions, links to Github, Patreon, etc.

**Comment by ArturWroblewski** [comment]
@aha999 Are you good at graphics, layout, photoshop? I am rather average in these subjects.

Anyone have any domain name suggestions?
What do you think about these below?

heroiclauncher.com

heroiclauncher.org

heroiclauncher.net

@flavioislima I understand you don't have time for this.

But would it be wrong for other people to do it for you? For sure someone will be here who has experience and could do it.

The only thing that is the minimum of your interference is choosing the domain name because it is your program. A domain can always be connected to another site in 15 minutes.

I can reserve the domain for myself, so as not to bother your head and then if you want, it will be transferred to you when you decide that you already have time for it.

**Comment by flavioislima** [comment]
> @aha999 Are you good at graphics, layout, photoshop? I am rather average in these subjects.
> 
> Anyone have any domain name suggestions?
> What do you think about these below?
> 
> heroiclauncher.com
> 
> heroiclauncher.org
> 
> heroiclauncher.net
> 
> @flavioislima I understand you don't have time for this.
> 
> But would it be wrong for other people to do it for you? For sure someone will be here who has experience and could do it.
> 
> The only thing that is the minimum of your interference is choosing the domain name because it is your program. A domain can always be connected to another site in 15 minutes.
> 
> I can reserve the domain for myself, so as not to bother your head and then if you want, it will be transferred to you when you decide that you already have time for it.

No, I do not bother if you guys deal with it.
About the domain, heroiclauncher.com sounds good to me :)

**Comment by ArturWroblewski** [comment]
I am writing this for information only.

We have a reserved domain of heroiclauncher.com
If you want me to transfer it to you, @flavioislima  I can do it only after 60 days. Vendor Restriction. Google Domains.

**Comment by flavioislima** [comment]
> I am writing this for information only.
> 
> We have a reserved domain of heroiclauncher.com
> If you want me to transfer it to you, @flavioislima I can do it only after 60 days. Vendor Restriction. Google Domains.

OK, thanks. Sounds good to me. 👍

**Comment by dragonDScript** [comment]
I am decent at UI/UX. Is there a flutter design for this already?

Domain should be simple enough, so heroic.org, heroic.com, heroic-launcher.com or heroic-launcher.org.

Using heroic.launcher.TPD (where tpd is .com, .org) is also a good idea.

**Comment by ArturWroblewski** [comment]
@dragonDScript 
We have the domain heroiclauncher.com

I suggest you use sites.google.com.

It is free for such a simple site.
There is a WYSIWYG Editor
He has the option to use his css. And the html code.

I suggest you use sites.google.com.

It is free for such a simple site.
There is a WYSIWYG Editor
He has the option to use his css. And the html code.

I wanted to add a Game tab, which will have covers of games that are heroic tested and run on linux. As it looks in the heroic / epic store. Such quick and simple information on how many things work is definitely useful and suggests installation.

when I create a website and link to a domain, I will add you as a collaborator. For that, however, I would need your google email. I should set up a website within 2 days.

Maybe I could use better / faster communication than github. some chat.

**Comment by dragonDScript** [comment]
> @dragonDScript
> We have the domain heroiclauncher.com
> 
> I suggest you use sites.google.com.
> 
> It is free for such a simple site.
> There is a WYSIWYG Editor
> He has the option to use his css. And the html code.
> 
> I wanted to add a Game tab, which will have covers of games that are heroic tested and run on linux. As it looks in the heroic / epic store. Such quick and simple information on how many things work is definitely useful and suggests installation.
> 
> when I create a website and link to a domain, I will add you as a collaborator. For that, however, I would need your google email. I should set up a website within 2 days.
> 
> Maybe I could use better / faster communication than github. some chat.

Google Sites doesn't look professional. I suggest Jekyll or React.

**Comment by flavioislima** [comment]
React + NextJS would be ideal since NEXT has amazing SEO support.

**Comment by flavioislima** [comment]
@aha999 
@ArturWroblewski 
I've created and deployed a simple website, ofc we need to add more information, a downloads page, etc. But the repository is open source so any of you can improve it.

Right now I'm using a temporary URL: https://heroic-website.vercel.app
But we can change to the domain @ArturWroblewski got.
The repository is this one: https://github.com/Heroic-Games-Launcher/heroic-website

**Comment by dragonDScript** [comment]
I'll work on the downloads section.
